virtual/Vagrantfile-ubuntu

```diff
@@ -32,7 +32,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       v.cpu_mode = "host-passthrough"
       # comment in for pci passthrough (and change bus according
       # to local hw setup - `lspci -nnk | grep NVIDIA`)
-      #v.pci :bus => '0x08', :slot => '0x00', :function => '0x0'
+      #BUS-GPU01 v.pci :bus => '0x08', :slot => '0x00', :function => '0x0'
```

Comment:
Can we add a comment here that somehow indicates that BUS-GPU01 is a magic string for the Jenkins builds?

This is mostly to avoid confusing anyone who is using the virtual cluster directly. It's frequently a new user's first experiment with DeepOps, and a lot of people (like me!) use it in their development workflows.